### PR TITLE
added vtol state implementation (follow up)

### PR DESCRIPTION
**Pull request by mattes-bru**
vtol state implementation in the telemetry plugin

### CODE REVIEW

**Comment by julianoes** [comment]
The proto change was already done but this we might be able to use, thank you!

**Comment by mattes-bru** [comment]
Yes, I saw that @rligocki was first on the proto but for the implementation I don't see a pull request. Any chance to merge his proto and my implementation?

File: src/plugins/telemetry/include/plugins/telemetry/telemetry.h [hunk 1]
```diff
@@ -148,6 +148,24 @@ class Telemetry : public PluginBase {
      */
     friend std::ostream& operator<<(std::ostream& str, Telemetry::LandedState const& landed_state);
 
+    /**
+     * @brief VTOL State enumeration
+     */
+    enum class VtolState {
```
Comment (mattes-bru) [inline src/plugins/telemetry/include/plugins/telemetry/telemetry.h:154]: I used the same naming scheme like in the other enums although I would personally prefer using more intuitive names than Mc and Fw

**Comment by rligocki** [comment]
Pull request should be already done on Proto side. I think It was today when @julianoes merged it.

**Comment by julianoes** [comment]
> Yes, I saw that @rligocki was first on the proto but for the implementation I don't see a pull request. Any chance to merge his proto and my implementation?

Let's try.

**Comment by julianoes** [comment]
I've force-pushed this with the latest changes from https://github.com/mavlink/MAVSDK-Proto/pull/251.

**Review by julianoes — APPROVED** [review]

File: src/plugins/telemetry/include/plugins/telemetry/telemetry.h (hunk 1 above)
Comment (bazfp) [reply src/plugins/telemetry/include/plugins/telemetry/telemetry.h:154]: Yes the enums are just based on MAVLink definitions https://mavlink.io/en/messages/common.html#MAV_VTOL_STATE

**Comment by bazfp** [comment]
Looks good, should work with `telemetry_server` (I should make some tests :cry: )

**Pull request merged**